packages/core/src/modules/routing/__tests__/pickup.test.ts

```diff
@@ -5,7 +5,7 @@ import { Subject } from 'rxjs'
 
 import { SubjectInboundTransport } from '../../../../../../tests/transport/SubjectInboundTransport'
```

Comment:
They don't...

I made a change to set the default key derivation to RAW to speed up wallet opening and closing. I should have probably done this in a separate PR. But funnily enough that broke some tests because they're quicker now (and thus certain events happened that shouldn't happen, and weren't happening before)